### DISCUSSION TITLE
The public schema for this project is not exposed

### DISCUSSION THREAD

**Discussion by AmineGuitouni** [supabase/supabase#30337] [Questions]
i self hosted supabase on azure vm using coolify. i did not change any thing yet but when i enter to the dashboard and go the api section i get this error message.

![image](https://github.com/user-attachments/assets/2d73e038-1b1e-40d2-adf0-f711246e75c6)

![image](https://github.com/user-attachments/assets/805cdc93-0ca9-4066-8e17-815a53b55e60)

i tried to open every port in my vm but didn't work
![image](https://github.com/user-attachments/assets/7a324e8b-9101-41ce-a7ab-66c77126edd9)

**Comment by Manikantaborra** [comment]
@AmineGuitouni

Did you find the solution? I'm also struggling with same issue :(
